src/pages/resources/testnet/upgrades/v35-3.mdx

```diff
@@ -1,8 +1,8 @@
-# Testnet upgrade: v0.35
+# Testnet upgrade: v0.35.3
```

Comment:
```suggestion
# Testnet upgrade: v0.35
```
The patch version doesn't need to be mentioned in the title, similar to past upgrades. Makes it seem like all patch releases are consensus breaking